WcaOnRails/app/models/competition.rb

```diff
@@ -156,6 +156,59 @@ def user_cannot_demote_themself
     end
   end
 
+  attr_accessor :receive_registration_emails
+  def receive_registration_emails=(r)
+    @receive_registration_emails = ActiveRecord::Type::Boolean.new.type_cast_from_database(r)
```

Comment:
Could you leave a comment why this is necessary? I never had to use `type_cast_from_database` manually yet. Is it an issue with MySQL?